plaso/parsers/winevtx.py

```diff
@@ -36,6 +36,7 @@ class WinEvtxRecordEventData(events.EventData):
     strings_parsed ([dict]): parsed information from event strings.
     user_sid (str): user security identifier (SID) stored in the event record.
     xml_string (str): XML representation of the event.
+    xml ([dict]): parsed information from XML string.
```

Comment:
Per conversation, since this is kibana specific move this into an output module instead.